src/workloads/execution/PipelineUpdate.yml

```diff
@@ -0,0 +1,102 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 10000
+    BatchSize: 2000
+    Document: &Document
+      t: &partitionID {^RandomInt: {min: 0, max: 100}}
+      randomID: &randomID {^RandomInt: {min: 0, max: 10000}}
+      int: &integer {^RandomInt: {min: 0, max: 100}}
+      str: &string {^RandomString: {length: {^RandomInt: {min: 20, max: 50}}}}
+      largeStr: &largeString {^RandomString: {length: 1024}}
+      obj: &object {a: *integer, b: *integer, c: *string}
+      arr: &array [*integer, *string, *object, *largeString]
+      nestedObj: &nestedObject {
+        int: *integer,
+        str: *string,
+        obj: *object,
+        arr: [1, 2, 3],
+        nestedArr: [*integer, *string, *object, *array],
+        largeStr: *largeString}
+    Indexes:
+    - keys: {t: 1}
+  - &Nop {Nop: true}
+  - *Nop
+
+- Name: RunUpdates
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1000    # Classic updates phase.
+    Database: *db
+    Operations:
+    - OperationMetricsName: ClassicUpdateAddFields
+      OperationName: RunCommand
+      OperationCommand:
+        update: Collection0
+        updates: [{q: {t: *partitionID}, u:
```

Comment:
If I understand correctly, this will change the value of the 'int' field and add a new 'string' field. Suggest forking this into two tests: one about changing an existing field and one about adding a new field. For the one with a new field suggest giving it a really obvious name like "newStringField" or similar.